sushy/interop_profiles/ServiceRoot.v1_0_0.json

```diff
@@ -0,0 +1,73 @@
+{
+	"SchemaDefinition": "RedfishInteroperabilityProfile.v1_0_0",
+	"ProfileName": "RedfishProfile",
+	"ProfileVersion": "1.0.0",
+	"Purpose": "Redfish Interoperability profile",
+	"OwningEntity": "Dell",
+	"ContactInfo": "abc@dell.com",
```

Comment:
If this accepts multiple e-mail addresses, please list yours, Dhuldev's, Digambar's and mine. Mine is richard.pioso@dell.com. Later, we may want to consider adding the e-mail addresses of others who contributed to this.

If only one (1) e-mail address is supported, please feel free to use mine.

Reply:
You can use multiple email addresses in there as long as it's one string (comma separated would probably work).

Reply:
Added email ids of Richard, Digambar and myself.